pkg/controller/gc_controller.go

```diff
@@ -130,12 +137,43 @@ func (c *gcController) processQueueItem(key string) error {
 		return nil
 	}
 
-	log.Info("Backup has expired. Creating a DeleteBackupRequest.")
+	log.Info("Backup has expired")
 
-	req := pkgbackup.NewDeleteBackupRequest(backup.Name, string(backup.UID))
+	selector := labels.SelectorFromSet(labels.Set(map[string]string{
+		arkv1api.BackupNameLabel: backup.Name,
+		arkv1api.BackupUIDLabel:  string(backup.UID),
+	}))
 
-	_, err = c.deleteBackupRequestClient.DeleteBackupRequests(ns).Create(req)
+	dbrs, err := c.deleteBackupRequestLister.DeleteBackupRequests(ns).List(selector)
 	if err != nil {
+		return errors.Wrap(err, "error listing existing DeleteBackupRequests for backup")
+	}
+
+	// if there's an existing unprocessed deletion request for this backup, don't create
```

Comment:
I'm not sure if the cases where we've been seeing a flood of DBRs have actually been processed; they're getting generated and not processed, which I think is the problem.

I requested a copy of a DBR from one of the failing clusters to double check this.

Reply:
Looks like I'm incorrect; the duplicates being created do end up being processed.

Reply:
The case where they weren't processed was because the controller hadn't started because of the missing CRDs, which has been addressed. Additionally, this change won't create any additional DBRs if an unprocessed one exists.